### ISSUE TITLE
Pill should cost only 1 gold

### ISSUE THREAD

**Issue by jimkaal** [manny405/sapai#59]
Pill costs 3 gold
```
def test_pill_1gold(self):
    player = Player(shop=Shop(["sleeping-pill"]), team=Team(["fish"]))
    player.buy_food(0, 0)
    self.assertEqual(player.gold, 9)
```

**Comment by alexdriedger** [comment]
This can be closed since it was addressed in #61

**Comment by jimkaal** [comment]
fixed in #61
